### PR TITLE
#7711: Update hard-alike ops sweep config and doc

### PR DESCRIPTION
**Pull request by mcw-anasuya**
### Ticket
#7711  

### Problem description
Documentation incomplete for Hard-alike ops with supported params

### List of Ops
- [x] hardtanh
- [x] hardsigmoid
- [x] hardswish
- [x] hardshrink

### What's changed
Updated doc and corresponding sweep test files

### Checklist
- [x] [Post commit CI passes](https://github.com/tenstorrent/tt-metal/actions/runs/11178340548) - PASSED
- [x] Run Sweeps - PASSED

### CODE REVIEW

**Review by Aswinmcw — APPROVED** [review]
LGTM

**Review by KalaivaniMCW — APPROVED** [review]

**Review by VirdhatchaniKN — APPROVED** [review]
LGTM

File: ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp [hunk 1]
```diff
@@ -1018,7 +1020,7 @@ template <typename unary_operation_t>
 void bind_unary_composite_float_with_default(py::module& module, const unary_operation_t& operation, const std::string& parameter_name_a, const std::string& parameter_a_doc, float parameter_a_value, const std::string& description) {
     auto doc = fmt::format(
```
Comment (umadevimcw) [inline ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp:1021]: Change the description to note/info (to match the naming with the param we are passing).

File: ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp [hunk 2]
```diff
@@ -1030,13 +1032,7 @@ void bind_unary_composite_float_with_default(py::module& module, const unary_ope
...
             ttnn.Tensor: the output tensor.
 
-        Supported dtypes and layouts:
-
-        +----------------------------+---------------------------------+-------------------+
-        |     Dtypes                 |         Layouts                 |     Ranks         |
-        +----------------------------+---------------------------------+-------------------+
-        |    BFLOAT16                |       TILE                      |      2, 3, 4      |
-        +----------------------------+---------------------------------+-------------------+
+        {5}
```
Comment (umadevimcw) [inline ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp:1035]: Also, add the `Note` heading here and just pass the information alone from the function call.

File: ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp [hunk 3]
```diff
@@ -1450,19 +1446,43 @@ void py_module(py::module& module) {
...
+            +----------------------------+---------------------------------+-------------------+
+            |    BFLOAT16                |          TILE                   |      2, 3, 4      |
+            +----------------------------+---------------------------------+-------------------+)doc");
     detail::bind_unary_composite_floats_with_default(
         module,
         ttnn::hardtanh,
         "min", "min value", -1.0f,
         "max", "max value", 1.0f,
-        R"doc(Performs hardtanh function on :attr:`input_tensor`, :attr:`min`, :attr:`max`.)doc");
+        R"doc(
```
Comment (umadevimcw) [inline ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp:1477]: Just send the note information string alone, remove `Note`

**Review by umadevimcw — CHANGES_REQUESTED** [review]

File: ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp (hunk 3 above)
Comment (mcw-anasuya) [reply ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp:1478]: If I add note in the information string, I could not get the table inside the blue box as @yan-zaretskiy suggested. That's why I kept it there.

**Review by umadevimcw — APPROVED** [review]

**Review by eyonland — APPROVED** [review]

**Review by jdesousa-TT — APPROVED** [review]

**Review by eyonland — APPROVED** [review]

**Pull request merged**